src/main/webapp/resources/js/pages/projects/ncbi/create/CreateNcbiExport.tsx

```diff
@@ -0,0 +1,443 @@
+    samples: samplesResponse,
+    platforms,
+    strategies,
+    sources,
+    selections,
+  }));
+}
+
+/**
+ * React component to render a form for created a new NCBI SRA Export
+ * @constructor
+ */
+function CreateNcbiExport(): JSX.Element {
+  const { projectId } = useParams();
+  const { samples: originalSamples }: LoaderValues = useLoaderData();
+
+  const [form] = Form.useForm();
+  const navigate = useNavigate();
+
+  const [samples, setSamples] = React.useState<Record<string, SampleRecord>>(
+    {}
+  );
+  const [formSamples, setFormSamples] = React.useState<FormSamples>({});
+  const [invalid, setInvalid] = React.useState<SampleRecord[]>([]);
+  const [createStatus, setCreateStatus] = React.useState<CreateStatus>(
+    CreateStatus.IDLE
+  );
+
+  React.useEffect(() => {
+    const formValues: FormSamples = {};
+    const invalidSamples: SampleRecord[] = [];
+    const goodSamples: Record<string, SampleRecord> = {};
+    originalSamples.forEach((sample) => {
+      if (validateSample(sample)) {
+        formValues[sample.name] = {
+          bioSample: "",
+          libraryName: sample.name,
+          name: sample.name,
+          id: sample.id,
+          instrumentModel: { value: "", fromDefault: true },
+          libraryConstructionProtocol: { value: "", fromDefault: true },
+          librarySelection: { value: "", fromDefault: true },
+          librarySource: { value: "", fromDefault: true },
+          libraryStrategy: { value: "", fromDefault: true },
+          singles: [],
+          pairs: [],
+        };
+        // Only keep a reference to the good files.
+        goodSamples[sample.name] = sample;
+      } else {
+        invalidSamples.push(sample);
+      }
+    });
+    setFormSamples(formValues);
+    setSamples(goodSamples);
+    setInvalid(invalidSamples);
+  }, [originalSamples]);
+
+  /**
+   * Update the default value for each sample in the form
+   * @param field the field to update
+   * @param value new value
+   */
+  const updateDefaultValue: UpdateDefaultValues = (field, value): void => {
+    // Update all the samples that don't currently have a value set by the sample.
+    const values: FormSamples = form.getFieldValue("samples");
+
+    Object.values(values).forEach((sample) => {
+      if (sample[field].fromDefault || sample[field].value.length === 0) {
+        form.setFieldsValue({
+          samples: { [sample.name]: { [field]: { value, fromDefault: true } } },
+        });
+      }
+    });
+  };
+
+  const removeInvalidSamples = () => setInvalid([]);
+
+  /*
+    This prevents the release date to be in the past.
+     */
+  const disabledDate: RangePickerProps["disabledDate"] = (date): boolean => {
+    // Can not select days before today for release
+    return date && date < moment().startOf("day");
+  };
+
+  /**
+   * When submitting the form, validate that all fields are valid, format the data
+   * and submitted to the server.
```

Comment:
Typo changed `submitted` to `submit

Reply:
Updated in 1cc40c0a533bd2710735e9cf10dde66566d59928